src/shared/pager.c

```diff
@@ -161,8 +161,8 @@ void pager_close(void) {
                 return;
 
         /* Inform pager that we are done */
-        stdout = safe_fclose(stdout);
-        stderr = safe_fclose(stderr);
+        safe_fclose(stdout);
+        safe_fclose(stderr);
```

Comment:
Actually it's not "invalidated", just closed. So writes will fail, but the object is safe to access.

I did some digging, and 
https://www.gnu.org/software/libc/manual/html_node/Standard-Streams.html#Standard-Streams says that:
> In the GNU C Library, stdin, stdout, and stderr are normal variables which you can set just like any others. For example, to redirect the standard output to a file, you could do:
```C
fclose (stdout);
stdout = fopen ("standard-output-file", "w");
```
But that has no effect, it seems that the manual is wrong!
In fact looking at printf.c in glibc, it should crash as soon as `stdout == NULL`, but it doesn't, which suggests that the real `stdout` is wrapped by some macro.

So that patch seems correct: since setting `stdout` has no effect, we shouldn't set it.